en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md

```diff
@@ -1,359 +1,104 @@
 # API-M Distributed Deployment - Overview
 
-Before understanding how to deploy WSO2 API Manager (WSO2 API-M), let's understand the WSO2 API-M distributed deployment better.
+Before deploying WSO2 API Manager (WSO2 API-M), let's understand how the WSO2 API-M distributed deployment works.
+According to the recommended [deployment patterns]({{base_path}}/install-and-setup/setup/deployment-overview/#simple-scalable-deployment), a distributed deployment includes API-M server nodes that separately run the [API-M profiles]({{base_path}}/install-and-setup/setup/distributed-deployment/product-profiles). An API-M profile is a runtime instance that only runs specific components of the runtime.
 
-## API-M Components
-
-WSO2 API Manager uses the following main components:
+## API-M Profiles
 
-<a href="{{base_path}}/assets/img/setup-and-install/api-m-components.png"><img src="{{base_path}}/assets/img/setup-and-install/api-m-components.png" alt="api-m components"></a>
+Listed below are the different profiles available in WSO2 API Manager. 
 
 <table>
     <tr>
         <th>
-            Component
-        </th>
+            Profile
+        </td>
         <th>
             Description
-        </th>
-    </tr>
-    <tr>
-        <td>
-            Gateway
-        </td>
-        <td>
-            Responsible for securing, protecting, managing, and scaling API calls.
         </td>
     </tr>
     <tr>
         <td>
-            Key Manager
+            Gateway Profile
         </td>
         <td>
-            Responsible for all security and key-related operations.
+            <p>Only starts the components related to the API Gateway.</p>
+<p>Use this when the API Gateway acts as a worker node in a cluster. This profile starts the backend features for data processing and communicates with the management node.</p>
```

Comment:
@tharindu1st , please verify this description. Is it correct to use the terminology "worker node" here?

Reply:
lets remove worker node concept from docs.

Reply:
@nilminiwso2 , please note

Reply:
fixed